Sources/Fullscreen/Receipt/ReceiptView.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - LoginViewDelegatew
+
+public protocol ReceiptViewDelegate: class {
+    func receipt(_ : ReceiptView, didTapNavigateToAd button: Button)
+    func receipt(_ : ReceiptView, didTapNavigateToMyAds button: Button)
+    func receipt(_ : ReceiptView, didTapCreateNewAd button: Button)
```

Comment:
Just a question, but do you need references to the button that was tapped? Or could it be:

```swift
public protocol ReceiptViewDelegate: class {
    func receiptDidTapNavigateToAd(_ : ReceiptView)
    func receiptDidTapNavigateToMyAds(_ : ReceiptView)
    func receiptDidTapCreateNewAd(_ : ReceiptView)
    func receiptInsertViewBelowDetailText(_ : ReceiptView) -> UIView?
}
```

Reply:
Not really, just for consistency with our other interfaces and Apple APIs.